### PR TITLE
lesson02 submission

### PR DESCRIPTION
**Pull request by mimalvarez**
Submitting my lesson02.

### CODE REVIEW

File: students/mimalvarez_pintor/lesson02/FizzBuzz.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+# Fizz Buzz Problem 
+for value in range(1,101):
+    if value % 3 == 0 and value % 5 == 0:
+        print('FizzBuzz')    
+    elif value % 3 == 0:
+            print('Fizz')
+    elif value % 5 == 0:
+            print('Buzz')
+    else:
+        print(value)
```
Comment (charlesrobison) [inline students/mimalvarez_pintor/lesson02/FizzBuzz.py:17]: This works - nice job

File: students/mimalvarez_pintor/lesson02/Grid_Printer.py [hunk 2]
```diff
@@ -0,0 +1,85 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Mon Apr 13 21:42:59 2020
+
+@author: miriam
+"""
+# Printing a grid part 1
+n= 1
```
Comment (charlesrobison) [inline students/mimalvarez_pintor/lesson02/Grid_Printer.py:8]: Missing space before assignment operator: n = 1, here and elsewhere in this code

File: students/mimalvarez_pintor/lesson02/Grid_Printer.py [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+m= 4
+
+plus= '+'
+minus= ' -'
+wall='|'
+space= '  '
+def line():
+    print(n*plus,m*minus,n*plus, m*minus,n*plus)
```
Comment (charlesrobison) [inline students/mimalvarez_pintor/lesson02/Grid_Printer.py:16]: You'll want spaces after the commas

File: students/mimalvarez_pintor/lesson02/Grid_Printer.py [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+minus= '-'
+wall='|'
+space= ' '
+def print_grid(l):
+    Newl=l+2
+    for j in range(Newl):
+        if j == 0 or j == int(Newl/2) or j == Newl -1:
+            for i in range(Newl):
+                if i == 0 or i == int(Newl/2):
+                    print (plus, end = space)
```
Comment (charlesrobison) [inline students/mimalvarez_pintor/lesson02/Grid_Printer.py:39]: No spaces after function names:  print(
Also, Within these parentheses you don't want spaces around the assignment operator:  print(plus, end=space).

File: students/mimalvarez_pintor/lesson02/series.py [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+"""
+Created on Wed Apr 15 22:36:54 2020
+
+@author: miriam
+"""
+
+# Fibonacci Series Exercise
+def fibonacci(n):
+    """Computes the nth value in the fibonacci series starting with zero index."""
+    if n==0:
```
Comment (charlesrobison) [inline students/mimalvarez_pintor/lesson02/series.py:11]: Spaces needed around equals signs: n == 0

File: students/mimalvarez_pintor/lesson02/series.py [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+        return 2
+    elif n==1:
+        return 1
+    else: 
+        return lucas(n-1) + lucas(n-2)
+    
+    pass
+
+# Generalizing
+def sum_series(n,n0=0, n1=1):
```
Comment (charlesrobison) [inline students/mimalvarez_pintor/lesson02/series.py:32]: Spaces needed after commas here

File: students/mimalvarez_pintor/lesson02/series.py [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+
+    # test if sum_series works for arbitrary initial values
+    assert sum_series(0, 3, 2) == 3
+    assert sum_series(1, 3, 2) == 2
+    assert sum_series(2, 3, 2) == 5
+    assert sum_series(3, 3, 2) == 7
+    assert sum_series(4, 3, 2) == 12
+    assert sum_series(5, 3, 2) == 19
+
+    print("tests passed")
```
Comment (charlesrobison) [inline students/mimalvarez_pintor/lesson02/series.py:78]: Tests look good!

**Comment by mimalvarez** [comment]
Great, thank you!

**Pull request merged**